src/jit/codegen.h

```diff
@@ -15,7 +15,7 @@
 #include "regset.h"
 #include "jitgcinfo.h"
 
-#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_) || defined(_TARGET_ARM_)
+#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_) || (defined(_TARGET_ARM_)&& !defined(ARM_SOFTFP))
```

Comment:
The real cause for that change was that regState->rsCalleeRegArgMaskLiveIn had some garbage inside unless the function raSetupArgMasks was called for the regState